### PR TITLE
All go-ethereum imports removed.

### PR DESCRIPTION
**Pull request by philip-morlier**
Flume as well the polygon plugin build without errors. The tests in api and indexing packages still need to be addressed.

### CODE REVIEW

File: api/types.go [hunk 1]
```diff
@@ -68,62 +71,62 @@ type paginator[T any] struct {
...
-	Sha3Uncles       common.Hash      `json:"sha3Uncles"`
-	Size             hexutil.Uint64   `json:"size"`
-	StateRoot        common.Hash      `json:"stateRoot"`
-	Timestamp        hexutil.Uint64   `json:"timeStamp"`
-	TotalDifficulty  *hexutil.Big     `json:"totalDifficulty"`
-	Transactions     interface{}      `json:"transactiions"`
-	TransactionsRoot common.Hash      `json:"transactionsRoot"`
-	Uncles           []common.Hash    `json:"uncles"`
-}
+// type Block struct {
```
Comment (AusIV) [inline api/types.go]: Let's remove this if we're not using it.

File: indexer/indexer.go [hunk 2]
```diff
@@ -136,8 +139,8 @@ func ApplyParameters(query string, params ...interface{}) string {
 			}
 		case hexutil.Uint64:
 			preparedParams[i] = fmt.Sprintf("%v", uint64(value))
-		case types.BlockNonce:
-			preparedParams[i] = fmt.Sprintf("%v", int64(value.Uint64()))
+		case BlockNonce:
```
Comment (AusIV) [inline indexer/indexer.go:142]: Didn't we end up not using this type?

File: indexer/types.go [hunk 3]
```diff
@@ -7,3 +7,5 @@ import (
 type Indexer interface {
 	Index(*delivery.PendingBatch) ([]string, error)
 }
+
+type BlockNonce int64
```
Comment (AusIV) [inline indexer/types.go:11]: Didn't we end up not using this type?

File: api/types.go (hunk 1 above)
Comment (philip-morlier) [reply api/types.go]: Done.

File: indexer/indexer.go (hunk 2 above)
Comment (philip-morlier) [reply indexer/indexer.go:142]: The indexer will be addressed in an upcoming PR.

File: indexer/types.go (hunk 3 above)
Comment (philip-morlier) [reply indexer/types.go:11]: Same as above.

**Pull request merged**